projects/angular/src/emphasis/alert/_alert.clarity.scss

```diff
@@ -489,9 +489,18 @@
   //1.cards
   .card {
     .alert {
-      $clr-alert-card-horizontal-margin: 0;
-      $clr-alert-card-vertical-margin: variables.$clr_baselineRem_0_25;
-      margin: $clr-alert-card-vertical-margin $clr-alert-card-horizontal-margin;
+      @include mixins.css-var(
+        margin-top,
+        clr-card-alert-margin-top,
+        variables.$clr_baselineRem_0_25,
+        variables.$clr-use-custom-properties
+      );
+      @include mixins.css-var(
+        margin-bottom,
+        clr-card-alert-margin-bottom,
+        variables.$clr_baselineRem_0_25,
+        variables.$clr-use-custom-properties
+      );
```

Comment:
Where did the horizontal margin go?
Now we have separate top and bottom and no left-right